code/game/objects/items/weapons/soap.dm

```diff
@@ -1,4 +1,4 @@
-/obj/item/soap
+code/modules/clothing/head/collectable.dm/obj/item/soap
```

Comment:
```suggestion
/obj/item/soap
```
I think this would be a more sane way to write this down

Reply:
I don't know why this is even here I hate it.